backends/s3/s3.go

```diff
@@ -103,7 +104,7 @@ type Backend struct {
 	log    logr.Logger
 
 	mu         sync.Mutex
-	lastMarker string
+	lastMarker []byte
```

Comment:
Making this a `[]byte` makes it harder to use and there is no reason this should not be a readable value.